ballerina/tests/Config.toml

```diff
@@ -16,6 +16,6 @@
 # under the License.
 #
 [ballerina.sql]
-maxOpenConnections=10
+maxOpenConnections=15
```

Comment:
Why this is increased?

Reply:
Revert back to 10 by ordering few test cases in different test groups.